src/odemis/gui/cont/stream.py

```diff
@@ -1386,12 +1388,33 @@ def __init__(self, stream_bar, stream, tab_data_model, show_panel=True, view=Non
         assert isinstance(stream, acqstream.SEMStream)  # don't show for CCD stream
         self._add_fastem_ctrls()
 
+    def _create_immersion_mode_cbox(self):
+        # HACK manually create the checkbox using custom gb_sizer's pos and span
```

Comment:
can you add to this comment why this hack is necessary?

Reply:
Comment added